### DISCUSSION TITLE
Menu sub-items for zooming

### DISCUSSION THREAD

**Discussion by FraserLee** [tauri-apps/tauri#4135] [General]
It seems like a few of the standard options (at least on mac) are missing. It might be nice to get the app-agnostic zoom shortcuts  supported to be able to do something like

```rust
let menu = Menu::new() // re-add macos defaults
    .add_submenu(Submenu::new("Edit", Menu::new()
        /* ... */
    ))
    .add_submenu(Submenu::new("View", Menu::new()
        .add_native_item(MenuItem::ZoomIn)
        .add_native_item(MenuItem::ZoomOut)
        .add_native_item(MenuItem::ZoomReset)
        .add_native_item(MenuItem::Separator)
        .add_native_item(MenuItem::ToggleFullscreen)
    ))
    .add_submenu(Submenu::new("Window", Menu::new()
        /* ... */
    ));
```
without binding them manually.

I'm really new to tauri - found this framework a few hours ago (& like it so far) - so it's more than possible there's some more correct api calls than what I've written.

**Comment by mike-luabase** [comment]
@FraserLee Did you ever figure this out? Trying to implement zooming in our app.
